### PR TITLE
Add Parascientific Escape Cruise in the Distant Seas

### PR DESCRIPTION
**Pull request by Demos25**
Adds Parascientific Escape Cruise in the Distant Seas.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/parascientific-escape-cruise-in-the-distant-seas/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "Parascientific Escape Cruise in the Distant Seas"
+description = "Hitomi Akeneno is a 16 year old high school student with a little secret - she has not one, but two psychic powers.\n\nAfter receiving an invitation, she boards a luxury cruise ship with her friend, Chisono Shio. As soon as she enters a suite room, a loud explosion sounds…and she finds herself stuck, unable to get out. Is this an accident? Or something far more sinister…?\n\nThe only way to find out is to escape, with the help of her powers."
```
Comment (TheKoopaKingdom) [inline games/parascientific-escape-cruise-in-the-distant-seas/game.dat]: The `\n`s here aren't needed.

File: games/parascientific-escape-cruise-in-the-distant-seas/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title = "0004000000186500"
+region = "usa"
+release_date = "2016-03-03"
+
+[[testcases]]
+title = "0004000000186500"
+compatibility = "0"
+date = "2017-10-10"
+version = "HEAD-e1bb198"
+cpu = "i5-3330"
```
Comment (TheKoopaKingdom) [inline games/parascientific-escape-cruise-in-the-distant-seas/game.dat]: Add `Intel Core` to the beginning here.

File: games/parascientific-escape-cruise-in-the-distant-seas/game.dat (hunk 1 above)
Comment (Leo626) [inline games/parascientific-escape-cruise-in-the-distant-seas/game.dat]: Don't really like the description. Use this one from [here](http://parascience.wikia.com/wiki/Parascientific_Escape:_Cruise_in_the_Distant_Seas):
```
Parascientific Escape Cruise in the Distant Seas is a visual novel adventure game developed by Intense, and the first entry in the Parascientific Escape trilogy. It was self-published in Japan and published by CIRCLE Ent. in North America and Europe.
```

File: games/parascientific-escape-cruise-in-the-distant-seas/game.dat [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+[[releases]]
+title = "0004000000186500"
+region = "usa"
+release_date = "2016-03-03"
+
+[[testcases]]
+title = "0004000000186500"
+compatibility = "0"
+date = "2017-10-10"
```
Comment (Leo626) [inline games/parascientific-escape-cruise-in-the-distant-seas/game.dat]: The testcase date is from over a week ago. It needs to be a current date.

File: games/parascientific-escape-cruise-in-the-distant-seas/game.dat (hunk 2 above)
Comment (Leo626) [inline games/parascientific-escape-cruise-in-the-distant-seas/game.dat]: `Intel`

File: games/parascientific-escape-cruise-in-the-distant-seas/game.dat [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+version = "HEAD-e1bb198"
+cpu = "i5-3330"
+gpu = "Radeon HD 6750"
```
Comment (Leo626) [inline games/parascientific-escape-cruise-in-the-distant-seas/game.dat]: `AMD`

**Comment by Demos25** [comment]
Writing a description, and search for a titleid for the each game is just stupid, it takes too much time and doesn't make sense for the end user who just want to find out if this game is working right now. moreover, about 1/6 of the games on the list have the wrong titleid.

And what's annoys me even more: You have a different requirements for publishing in the webform on the site and wiki on github
https://services.citra-emu.org/game/create/
For example, here says: "Screenshots give an idea of what the game should look while running. **You need at least one.** "

I have about 100 tested games, but I'm absolutely don't want to add them to the list now. This system is just pisses me off. I hope thst someday it will be redesigned in better way.

Btw: Sorry for my english, this's not my native language.

**Comment by Schplee** [comment]
@Demos25 Little of what we do is done directly for the user. The compatibility database is a functional archive of all the games' states, and how those change over time as Citra is updated and games are tested in conjunction with those new features.

As a byproduct, a database that users can refer to is created, one that allows those to refer to a variety of data and relevant information. When someone decides that they wish ton contribute, they are contributing to this archival process, not a quick "game works yes/no" page that is not only vague, but completely useless when a games functionality can range drastically. Just like our main codebase for Citra itself, we want to get it right the first time, and have strict quality control to ensure the aforementioned.

Now, in regards to your claim that 1/6th of the games have the wrong TitleID listed, it'd be useful if you were to generate a list of said entries and create an issue so that we may fix it.

Thank you for pointing out the discrepancy between the contributions guide and the online PR submitter, that will be fixed.

If you have any ideas on how the system should be changed to better suit the needs of contributors, feel free to create an issue detailing what is wrong, how it could be fixed, and in what ways the submission process could be easier.

**Comment by TheKoopaKingdom** [comment]
Closing this as it seems to have been abandoned by OP.

**Pull request closed without merging**